.github/workflows/pypi.yaml

```diff
@@ -0,0 +1,71 @@
+---
+name: Build, test, and upload PyPI package
```

Comment:
FTR the workflow doesn't do testing. The name is inaccurate.

Reply:
The workflow is testing package build and integrity with Hynek's action.